### PR TITLE
Project initialization

### PR DESCRIPTION
**Pull request by hancel-avila**
# Description
We installed the current dependencies that we're going to use on this project, such as I18n library, Formik, Yup, etc...
Why?
This will help us to start with complementary forms.

### CODE REVIEW

**Review by DemisRincon — APPROVED** [review]

File: PetRescue/src/components/CustomButton.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import {View} from 'react-native';
+
+const CustomButton = () => {
+  return <View></View>;
```
Comment (cfcortes) [inline PetRescue/src/components/CustomButton.js:5]: Codacy has a fix for the issue: [Empty components are self-closing](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7863066)
```suggestion
  return <View />;
```

File: PetRescue/src/containers/Login/index.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import {View} from 'react-native';
+
+const Login = () => {
+  return <View></View>;
```
Comment (cfcortes) [inline PetRescue/src/containers/Login/index.js:5]: Codacy has a fix for the issue: [Empty components are self-closing](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7863066)
```suggestion
  return <View />;
```

File: PetRescue/src/components/TextInput.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import {View} from 'react-native';
+
+const TextInput = () => {
+  return <View></View>;
```
Comment (cfcortes) [inline PetRescue/src/components/TextInput.js:5]: Codacy has a fix for the issue: [Empty components are self-closing](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7863066)
```suggestion
  return <View />;
```

File: PetRescue/src/containers/Login/index.js (hunk 2 above)
Comment (cfcortes) [inline PetRescue/src/containers/Login/index.js:5]: Codacy has a fix for the issue: [Empty components are self-closing](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7863066)
```suggestion
  return <View />;
```

File: PetRescue/src/components/TextInput.js (hunk 3 above)
Comment (cfcortes) [inline PetRescue/src/components/TextInput.js:5]: Codacy has a fix for the issue: [Empty components are self-closing](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7863066)
```suggestion
  return <View />;
```

File: PetRescue/src/components/CustomButton.js (hunk 1 above)
Comment (cfcortes) [inline PetRescue/src/components/CustomButton.js:5]: Codacy has a fix for the issue: [Empty components are self-closing](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7863066)
```suggestion
  return <View />;
```

**Pull request merged**